src/Symfony/Component/Serializer/Encoder/ChainDecoder.php

```diff
@@ -19,8 +19,9 @@
  * @author Jordi Boggiano <j.boggiano@seld.be>
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  * @author Lukas Kahwe Smith <smith@pooteeweet.org>
+ * @author Kévin Dunglas <dunglas@gmail.com>
  */
-class ChainDecoder implements DecoderInterface
+class ChainDecoder implements DecoderInterface/*, ContextAwareDecoderInterface*/
```

Comment:
Because it will introduce a BC break (it will change the signature of the public method of this class).